### PR TITLE
Fix typos of the 'utilities'

### PR DESCRIPTION
**Pull request by mooncak**
**What this PR does / why we need it**:
Fix typos of the 'utilities': 

1. pkg/kubectl/cmd/taint/utils.go
2. pkg/util/taints/taints.go

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/taint/utils.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package taints implements utilites for working with taints
+// Package taints implements utilities for working with taints
```
Comment (nikhita) [inline pkg/kubectl/cmd/taint/utils.go]: @mooncak thanks for the cleanup! Can you also fix this typo in this line:

https://github.com/kubernetes/kubernetes/blob/d20912c83fe3b9536098920ae279d2201a424c43/test/e2e/storage/persistent_volumes-local.go#L1293

**Comment by nikhita** [comment]
/kind cleanup

@mooncak Please also squash the commits into a single commit.

File: pkg/kubectl/cmd/taint/utils.go (hunk 1 above)
Comment (mooncak) [reply pkg/kubectl/cmd/taint/utils.go]: sure

Comment (mooncak) [reply pkg/kubectl/cmd/taint/utils.go]: @nikhita it's already merged.

Comment (nikhita) [reply pkg/kubectl/cmd/taint/utils.go]: > @nikhita it's already merged.

I don't follow.

> nginx has this utilites.

Here it should be s/utilites/utilities

Comment (mooncak) [reply pkg/kubectl/cmd/taint/utils.go]: > > @nikhita it's already merged.
> 
> I don't follow.
> 
> > nginx has this utilites.
> 
> Here it should be s/utilites/utilities

see #69784

Comment (nikhita) [reply pkg/kubectl/cmd/taint/utils.go]: @mooncak Can you make the change in https://github.com/kubernetes/kubernetes/pull/69784 in this PR itself? It's better to combine related cleanups together - easier for review + approval. :)

**Comment by mooncak** [comment]
> /kind cleanup
> 
> @mooncak Please also squash the commits into a single commit.

Track this issue by #69888

**Comment by mooncak** [comment]
Another PR #69888  tracked this issue, close this PR..

**Pull request closed without merging**